### DISCUSSION TITLE
Sector doesn't appear on original tag, how can I delete it from the target tag?

### DISCUSSION THREAD

**Discussion by IroEro** [ikarus23/MifareClassicTool#515] [General]
Hi, I have a tag which I want to duplicate. I created a dump of it, then wrote that dump to a gen2 tag. The key and the rest of the data was duplicated successfully, but for some reason the target tag has an additional sector 1 which the original doesn't seem to have (or at least doesn't display in the app).
Is it even possible to delete a sector in the middle like that, or is it just a sector without a proper key which was therefore skipped by the reader? I have a feeling this is what stops it from working properly in devices which compare the entire tag and not just the ID
![Screenshot_2024-12-02-11-00-12-36_95b7d1e8510fa3032a240bd8b4b39632](https://github.com/user-attachments/assets/5bcaa917-cd75-46eb-9fba-baf57174ebec)
![Screenshot_2024-12-02-10-59-00-67_95b7d1e8510fa3032a240bd8b4b39632](https://github.com/user-attachments/assets/0c3a5de2-6bb1-4577-b808-1bdbffe94030)

**Comment by ikarus23** [comment]
Hi. Sorry for the late response.

It seams like you did not have all keys when you read the original tag. If you don't have the keys to read sector 1, an error message will show that and the saved dump will not include that sector. There will be no error message when you load the dump. The sector is just missing.

When you write a dump with a missing sector (e.g. 1) then the sector is skipped. Therefore, on the target tag the sector 1 is untouched.

It is very likely, that all important information is in the sector 1. The other sectors that are present on you screenshots are empty.  If you want to clone the original tag, you have to read out sector 1 as well. You need the keys for that. There are three options here:

1.  You know the keys
2. The keys are already known and part of the extended standard keys file that comes with the app.
3. You recover/crack the keys. The internet is full on with information on how to do that. The easiest way is to get a Proxmark3 device. Your Android phone can not crack the keys.

**Reply by ikarus23** [reply]
Oh, and if you really want to delete a sector from a dump you have to do it manually. You can edit the dump files of MCT (.mct) with an normal text editor. Just export, edit and import them.

**Reply by IroEro** [reply]
Thank you so much for the detailed response!
A proxmark3 isn't an option at the moment, but maybe I'll dip my toes into cracking the key using something I have on hand, like an esp32.
